ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecoderOptions.java

```diff
@@ -55,9 +55,9 @@
 
   @Description("URI of a redis server that will be used for deduplication")
   @Validation.Required
-  ValueProvider<URI> getRedisUri();
+  ValueProvider<String> getRedisUri();
```

Comment:
It looks like Beam was passing this to Jackson; Jackson was expecting a JSON blob that it could deserialize into a URI, which isn't what we want.

So I changed this to String and we now have a separate parsing step.